packages/contentful-migrations/ui/1100000000006-create-m-editorial-migration.js

```diff
@@ -1,20 +1,16 @@
-module.exports = async function (migration, context) {
-  const { makeRequest } = context;
-  // Fetch locale
-  const { items: locales } = await makeRequest({
-    method: 'GET',
-    url: '/locales',
-  });
-  const defaultLocale = locales.find((locale) => locale.default);
+const { withHelpers } = require('@jungvonmatt/contentful-migrations');
+
+module.exports = withHelpers(async (migration, _context, helpers) => {
+  const defaultLocale = await helpers.locale.getDefaultLocale();
 
   const mEditorial = migration
     .createContentType('m-editorial')
```

Comment:
I still don't like the name `editorial` ;)
This is always very confusing and always needs to be explained with "something like a teaser"

Reply:
why not rename the editorial module to `teaser` even if teaser somehow implies that there's something to click and this module could also be used to just present a text alongside an image when the link is omitted.

Reply:
Yes, the name is not perfect. But I haven't found a better one yet. "Teaser" is even less appropriate in my opinion.  think we are trying to map too many use cases in one module/component... maybe. It saves content types, but it is not easier for the editor.

Reply:
Isn't it actually a classic hero unit? For example, we could name the module `m-hero-section` and the component `c-hero`. It is usually always a bolder text, optionally with CTA and media. Maybe also `hero-text`, or even more generic `hero-content`.

We could also consider using the component only internally in the templates. Then only `m-hero-section` would remain, which is actually quite understandable.

See Tailwind UI: https://tailwindui.com/components/marketing/sections/heroes

By the way, Bootstrap calls this type "Jumbotron" ... I don't really like that.

Reply:
I like `m-hero` eventually even without the `-section`. I would also drop the component reference as it' very unlikely that the content from the hero component/module will be reused somewhere else. It would be much simpler to just edit text/media/... directly in the module

Reply:
Done